### PR TITLE
JAV-89 grouping related modules together under parent modules

### PR DESCRIPTION
**Pull request by TankTian**

### CODE REVIEW

File: handler/handler-bizkeeper/pom.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
 		<groupId>io.servicecomb</groupId>
-		<artifactId>java-chassis</artifactId>
+		<artifactId>handler</artifactId>
```
Comment (seanyinx) [inline handler/handler-bizkeeper/pom.xml]: can we rename parent module from handler to handlers?

File: pom.xml [hunk 2]
```diff
@@ -93,45 +93,23 @@
...
-		<module>java-chassis-distribution</module>
-		<module>coverage-reports</module>
-	</modules>
+    <modules>
+        <module>parent</module>
+        <module>java-chassis-dependencies</module>
+        <module>common</module>
+        <module>foundation</module>
+        <module>handler</module>
+        <module>provider</module>
```
Comment (seanyinx) [inline pom.xml]: can we rename parent module from handler to handlers, provider -> providers, transport -> transports?

Comment (TankTian) [inline pom.xml]: ok

File: common/common-javassist/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+/bin/
```
Comment (seanyinx) [inline common/common-javassist/.gitignore]: why adding this?

Comment (TankTian) [inline common/common-javassist/.gitignore]: mistake,maybe eclipse auto generate.later ,i will remove it .

**Pull request merged**